.cicd/scripts/srw_ftest.sh

```diff
@@ -71,14 +71,29 @@ echo "DATA_LOCATION=${DATA_LOCATION}"
 sed "s|^task_get_extrn_ics:|task_get_extrn_ics:\n  EXTRN_MDL_SOURCE_BASEDIR_ICS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518|1" -i ush/config.yaml
 sed "s|^task_get_extrn_lbcs:|task_get_extrn_lbcs:\n  EXTRN_MDL_SOURCE_BASEDIR_LBCS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518|1" -i ush/config.yaml
 
+hpss_machines=( jet hera )
+
+# Use staged data for HPSS supported machines
+if [[ ${hpss_machines[@]} =~ ${platform,,} ]] ; then
+    sed 's|^task_get_extrn_ics:|task_get_extrn_ics:\n  USE_USER_STAGED_EXTRN_FILES: true|g' -i ush/config.yaml
+    sed 's|^task_get_extrn_lbcs:|task_get_extrn_lbcs:\n  USE_USER_STAGED_EXTRN_FILES: true|g' -i ush/config.yaml
+fi
```

Comment:
Please replace lines 74 - 80 with:

`sed 's|^platform:|platform:\n  EXTRN_MDL_DATA_STORES: disk|g' -i ush/config.yaml`

This is necessary because the `USE_USER_STAGED_EXTRN_FILES` variable will set the above in the `run_WE2E_tests.py` script.  Since the wrappers aren't being run by `run_WE2E_tests.py`, setting this variable does nothing.

```suggestion
sed 's|^platform:|platform:\n  EXTRN_MDL_DATA_STORES: disk|g' -i ush/config.yaml
```